include/votca/xtp/gwbse.h

```diff
@@ -124,7 +124,7 @@ namespace votca {
             bool _store_eh_interaction;
 
             // automatic scissors iteration
```

Comment:
Is there an option for just "scissors" iteration left? Otherwise, comment should be clarified.